packages/react-components/react-shared-contexts/src/OverridesContext/OverridesContext.ts

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+/**
+ * @internal
+ */
+export type OverridesContextValue = {
+  // should be InputProps['appearance'], but do we want to add the dependency?
+  inputDefaultAppearance?: 'outline' | 'underline' | 'filled-darker' | 'filled-lighter';
+};
+
+/**
+ * @internal
+ */
+export const OverridesContext = React.createContext<OverridesContextValue | undefined>(undefined);
+OverridesContext.displayName = 'OverridesContext';
```

Comment:
Do we need this?

Reply:
This is how the React tree currently looks like for the `FluentProvider`. I wanted to make it easier to inspect.
![image](https://user-images.githubusercontent.com/9615899/214050976-40b806f9-50ec-4909-b917-733c4f10b570.png)
wdyt?

Reply:
I see, but don't apply display names for contexts anywhere else. So, I suggest to remove it.

Reply:
reverted